x-pack/plugins/lens/public/indexpattern_datasource/operations/layer_helpers.ts

```diff
@@ -141,12 +155,16 @@ export function insertNewColumn({
 
   const baseOptions = { indexPattern, previousColumn: layer.columns[columnId] };
 
-  if (operationDefinition.input === 'none') {
+  if (operationDefinition.input === 'none' || operationDefinition.input === 'managedReference') {
     if (field) {
       throw new Error(`Can't create operation ${op} with the provided field ${field.name}`);
     }
+    if (operationDefinition.input === 'managedReference') {
```

Comment:
Is this actually a problem? It seems like creating new formula and transitioning works fine right?

Reply:
You're right, this can be removed.

Reply:
Yes I think this is some old logic never removed.